corehq/apps/commtrack/views.py

```diff
@@ -264,3 +262,56 @@ def mk_action(action):
         self.domain_object.commtrack_settings.save()
 
         return self.get(request, *args, **kwargs)
+
+
+class StockLevelsView(BaseCommTrackManageView):
+    urlname = 'stock_levels'
+    page_title = ugettext_noop("Stock Levels")
+    template_name = 'commtrack/manage/stock_levels.html'
+
+    def get_existing_stock_levels(self):
+        """
+        Returns a list of dicts of this form:
+        {
+            'loc_type': 'district,
+            'emergency_level': 0.5,
+            'understock_threshold': 1.5,
+            'overstock_threshold': 3,
+        }
+        """
+        return [{
+            'loc_type': loc_type.name,
+            'emergency_level': 0.5,
+            'understock_threshold': 1.5,
+            'overstock_threshold': 3,
```

Comment:
Is this just hard coded now because this isn't a finished PR? Or is this meant to be a default?